### ISSUE TITLE
Error [ERR_REQUIRE_ESM]: Must use import to load ES Module

### ISSUE THREAD

**Issue by zestzero** [spencermountain/spacetime#319]
```
Error [ERR_REQUIRE_ESM]: Must use import to load ES Module: /home/runner/work/.../node_modules/spacetime/builds/spacetime.js
require() of ES modules is not supported.
require() of /home/runner/work/.../node_modules/spacetime/builds/spacetime.js from /home/runner/work/.../src/utils/timeManager.ts is an ES module file as it is a .js file whose nearest parent package.json contains "type": "module" which defines all .js files in that package scope as ES modules.
Instead rename spacetime.js to end in .cjs, change the requiring code to use import(), or remove "type": "module" from /home/runner/work/.../node_modules/spacetime/package.json.

    at Module._extensions..js (internal/modules/cjs/loader.js:1015:13)
    at Object.require.extensions.<computed> [as .js] (/home/runner/work/.../node_modules/ts-node/src/index.ts:851:44)
    at Module.load (internal/modules/cjs/loader.js:863:32)
    at Function.Module._load (internal/modules/cjs/loader.js:708:14)
    at Module.require (internal/modules/cjs/loader.js:887:19)
    at require (internal/modules/cjs/helpers.js:74:18)
    at Object.<anonymous> (/home/runner/work/.../src/utils/timeManager.ts:1:1)
    at Module._compile (internal/modules/cjs/loader.js:999:30)
    at Module.m._compile (/home/runner/work/.../node_modules/ts-node/src/index.ts:858:23)
    at Module._extensions..js (internal/modules/cjs/loader.js:1027:10)
```

My guess would be the package doesn't contain `.cjs`

**Comment by spencermountain** [comment]
fair enough - fixed in 7.1.0
cheers

**Comment by zestzero** [comment]
Thank you 👍🏻
